web/httpserver.py

```diff
@@ -168,10 +168,13 @@ def runsimple(func, server_address=("0.0.0.0", 8080)):
     
     server = WSGIServer(server_address, func)
 
-    if server.ssl_adapter:
-        print("https://%s:%d/" % server_address)
+    if '/' in server_address[0]:
```

Comment:
Wouldn't it be better to use `server_address.startswith('/')`?

Reply:
@anandology i've got this variant from https://github.com/webpy/webpy/blob/master/web/net.py#L156 , but i won't mind to change it, if you want.

Thank you for fast review!